src/components/Container.js

```diff
@@ -24,7 +44,7 @@ export default function Container() {
           <Categories />
         </Route>
       </Switch>
-      <AddBookForm />
+      <AddBookForm add={submitBookToStore} />
     </>
   );
 }
```

Comment:
- [ ] The Book link doesn't render the list of the books, seems to be a broken link.
![cpp](https://user-images.githubusercontent.com/55635977/132891114-e78b26a6-24ce-4bcc-b554-0af7152a7f56.png)